File: 코딩 기초 트레이닝/배열 만들기1-p181901/배열 만들기1.py
# https://school.programmers.co.kr/learn/courses/30/lessons/181901

def solution(n, k):
    answer = [k]

    while answer[len(answer) -1] < n:
        answer.append(answer[len(answer) -1] + k)

    return answer[:-1] if answer[len(answer) -1] > n else answer

File: 코딩 기초 트레이닝/배열 만들기1-p181901/배열 만들기1.js
// https://school.programmers.co.kr/learn/courses/30/lessons/181901

function solution(n, k) {
    var answer = [k];


    while(answer[answer.length-1] < n) {
        answer.push(answer[answer.length-1] + k);
    }

    return answer[answer.length-1] > n ? answer.slice(0, answer.length-1) : answer;
}
